src/rez/tests/test_config.py

```diff
@@ -226,6 +226,19 @@ def _data(self):
         finally:
             os.environ = old_environ
 
+    def test_7(self):
+        """Test path list environment variable with whitespace."""
+        c = Config([self.root_config_file], locked=False)
+
+        # test basic env-var override
+        packages_path = [
+            "c:/foo bar/baz",
+            "c:/foo 2 bar/baz",
+        ]
```

Comment:
You could add more samples to your test. For instance:
- /home/foo bar/baz
- /home/foo\ bar/baz
- c:\foo bar\baz

Reply:
done